api_tests/files/views/test_file_detail.py

```diff
@@ -71,6 +71,7 @@ def test_get_file(self):
         assert_equal(attributes['last_touched'], None)
         assert_equal(attributes['provider'], self.file.provider)
         assert_equal(attributes['size'], self.file.versions[-1].size)
+        assert_equal(attributes['current_version'], len(self.file.history))
```

Comment:
@pattisdr is this foolproof? Is there any way the file history count is not the latest version?